scripts/export-results.js

```diff
@@ -0,0 +1,51 @@
+const admin = require("firebase-admin")
+const fs = require("fs")
+require("dotenv").config({
+  path: `.env.development`,
+})
+
+const getFirebase = () => {
+  const firebasePrivateKey = process.env.FIREBASE_PRIVATE_KEY
+  if (!admin.apps.length) {
+    admin.initializeApp({
+      credential: admin.credential.cert({
```

Comment:
This object should be in in a global config file ad reused everywhere (DRY)

Reply:
I do have issue with netilfy function cant access file outcide api folder that why